packages/contracts/deploy/60_dao-factory.ts

```diff
@@ -21,7 +21,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
   const daoFactoryAddress: string = ret.receipt?.contractAddress || '';
 
-  const registerDAORole = ethers.utils.id('REGISTER_DAO_ROLE');
+  const registerDAOPermission = ethers.utils.id('REGISTER_DAO_ROLE');
```

Comment:
Shouldn't it be `_PERMISSION_ID` for consistency?

Reply:
Changed